pypeit/fluxcalibrate.py

```diff
@@ -57,20 +57,21 @@ def __init__(self, spec1dfiles, sensfiles, par=None, debug=False, outfiles=None)
             sobjs = specobjs.SpecObjs.from_fitsfile(spec1)
             history = History(sobjs.header)
             if sens != sens_last:
-                wave, zeropoint, meta_table, out_table, header_sens = sensfunc.SensFunc.load(sens)
+                s = sensfunc.SensFunc.from_file(sens)
+#                wave, zeropoint, meta_table, out_table, header_sens = sensfunc.SensFunc.load(sens)
                 history.append(f'PypeIt Flux calibration "{sens}"')
-            self.flux_calib(sobjs, wave, zeropoint, meta_table)
+            self.flux_calib(sobjs, s.wave, s.zeropoint, s.algorithm, s.sens)
```

Comment:
Given your changes, should it just be

self.flux_calib(sobjs, s)

?

Reply:
Good point.  I've updated the code to do exactly this.